### PR TITLE
Push resources at bulk

### PR DESCRIPTION
**Pull request by byawitz**
Deploy all resource at once by running
```sh
appwrite push --all --yes
```
Or select resource group by running 
```sh
appwrite push
```
image

### CODE REVIEW

**Review by loks0n — APPROVED** [review]

File: templates/cli/lib/commands/deploy.js.twig [hunk 1]
```diff
@@ -906,6 +919,11 @@ const deployTeam = async ({ all, yes } = {}) => {
         }
     }
 }
+const deploy = new Command("deploy")
+    .description(commandDescriptions['deploy'])
+    .option(`--all`, `Flag to deploy all resources`)
+    .option(`--yes`, `Flag to confirm all warnings`)
```
Comment (loks0n) [inline templates/cli/lib/commands/deploy.js.twig:925]: Does this include shorthand versions? e.g. `-y` and `-a`? They would be nice

File: templates/cli/lib/commands/deploy.js.twig [hunk 2]
```diff
@@ -906,6 +919,11 @@ const deployTeam = async ({ all, yes } = {}) => {
         }
     }
 }
+const deploy = new Command("deploy")
+    .description(commandDescriptions['deploy'])
```
Comment (christyjacob4) [inline templates/cli/lib/commands/deploy.js.twig:923]: Have you added the command description for the deploy command?

File: templates/cli/lib/commands/deploy.js.twig [hunk 3]
```diff
@@ -227,14 +227,27 @@ const awaitPools = {
...
-    .action(actionRunner(async (_options, command) => {
-        command.help()
-    }));
+const deployResources = async ({ all, yes } = {}) => {
+    const actions = {
+        functions: deployFunction,
+        collections: deployCollection,
+        buckets: deployBucket,
+        teams: deployTeam,
+        messages: new Function()
```
Comment (christyjacob4) [inline templates/cli/lib/commands/deploy.js.twig:236]: what exactly is this?

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Looks like the deployMessages function has not been implemented yet

File: templates/cli/lib/commands/deploy.js.twig (hunk 3 above)
Comment (byawitz) [reply templates/cli/lib/commands/deploy.js.twig]: The base branch has been updated to match the one that includes messaging implementation

File: templates/cli/lib/questions.js.twig [hunk 4]
```diff
@@ -277,6 +277,21 @@ const questionsLogin = [
...
+    {
+        type: "checkbox",
+        name: "resources",
+        message: "Which resources would you like to push?",
+        choices: [
+            { name: 'Functions', value: 'functions' },
+            { name: 'Collections', value: 'collections' },
+            { name: 'Buckets', value: 'buckets' },
+            { name: 'Teams', value: 'teams' },
+            { name: 'Messages', value: 'messages' }
```
Comment (Meldiron) [inline templates/cli/lib/questions.js.twig:290]: Let's do Topics here.

**Pull request merged**